### ISSUE TITLE
[week3] 오늘 요약 : hierarchical softmax, NCE,negative sampling, subsampling 좀더 생각해보자.

### ISSUE THREAD

**Issue by kimcando** [kimcando/BoostcampAITech3-PaperReading-Embedding#18]
* hierarchical softmax 참고 : https://talbaumel.github.io/blog/softmax/
* NCE 참고 : https://github.com/Stonesjtu/Pytorch-NCE
* word2vec skip-gram negative sampling 참고: https://direction-f.tistory.com/29
* basic embedding model 참고 : https://github.com/graykode/nlp-tutorial
* word2vec genism : https://github.com/RaRe-Technologies/gensim-data

다만 데이터를 먹이면서 학습 제대로 되는 예시 찾기는 어려운 것 같네요..
